ckan/lib/munge.py

```diff
@@ -134,9 +77,8 @@ def munge_filename_legacy(filename):
     Upload.update_data_dict() because if this routine changes then group images
     uploaded previous to the change may not be viewable.
     '''
-    filename = substitute_ascii_equivalents(filename)
     filename = filename.strip()
-    filename = re.sub(r'[^a-zA-Z0-9.\- ]', '', filename).replace(' ', '-')
```

Comment:
the name and docstring of this function suggest we shouldn't change anything here. If we can remove this it would be great as a separate PR

Reply:
@wardi Do you suggest to keep using `substitute_ascii_equivalents` in this function for this PR?

Reply:
Right.  Removed.